_posts/2021-01-01-improving-genomic-accuracy-dv1-0.markdown

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "Improving the Accuracy of Genomic Analysis with DeepVariant 1.0"
+source: Google AI Blog
+date: 2020-09-18
```

Comment:
Could you use this date in the filename too?

Reply:
All set - good catch!